src/py/flwr/common/constant.py

```diff
@@ -31,3 +31,4 @@
     TRANSPORT_TYPE_GRPC_RERE,
     TRANSPORT_TYPE_REST,
 ]
+TRANSPORT_DEFAULT_TIMEOUT = 60
```

Comment:
Let's rename this one slightly:
```suggestion
TRANSPORT_TIMEOUT_DEFAULT = 60
```